front/src/index.tsx

```diff
@@ -21,7 +20,7 @@ root.render(
         <Route path="/" element={<App />} />
         <Route path="/signup" element={<SignUp />} />
         <Route path="/login" element={<LogIn />} />
-        <Route path="/list" element={<BoardMain />} />
+        <Route path="/list" element={<BoardList />} />
```

Comment:
```suggestion
        <Route path="/list" element={<BoardList />} />
```
list 화면 확인을 위해 route target 수정했습니다